resources/rococo-local.json

```diff
@@ -6,11 +6,10 @@
   "telemetryEndpoints": null,
   "protocolId": "dot",
   "properties": null,
-  "forkBlocks": null,
-  "badBlocks": null,
-  "lightSyncState": null,
   "consensusEngine": null,
   "codeSubstitutes": {},
+  "relay_chain": "rococo-local",
+  "para_id": 2000,
```

Comment:
This is the spec for the relay chain, not a parachain chain, so it shouldn't have these two fields.

Not sure why it was trying to require them.

Reply:
the build actually throws an error if they're not included.

Reply:
@dopatraman Which build fails for you? I pulled this branch and `make build`, `make lint`, and `make start-frequency` all work.

Reply:
@dopatraman  I dont think this is needed, its not used in any builds